### ISSUE TITLE
Swift 2 not supported

### ISSUE THREAD

**Issue by altvisual** [benmel/HafenCity#15]
Swift 2 isn't supported, build fails.  Using xcode to convert code to swift 2 produces several errors and warnings.

**Comment by benmel** [comment]
Yes currently it uses Swift 1.2, which means you need to use Xcode 6 to build. Eventually I'll convert it to Swift 2.
